mixins/jcustomer.yml

```
@@ -350,4 +350,5 @@ actions:
     - cmd [cp]: |-
         setenv=$(find /opt/jcustomer/jcustomer/bin -name setenv)
         echo 'APM_OPTS="-javaagent:'${JCUSTOMER_HOME}'/jcustomer/datadog/dd-java-agent.jar -Ddd.logs.injection=true -Ddd.service=jcustomer -Ddd.env='${env.envName}' -Ddd.profiling.enabled=true"' >> $setenv
-        echo '${DATADOG_APM_ENABLED:-true} && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
+        echo '[ -z ${DATADOG_APM_ENABLED} ] && DATADOG_APM_ENABLED=true' >> $setenv
+        echo '${DATADOG_APM_ENABLED} && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
```

```suggestion
        # this fucking cloudscripting interprets the bash form `${<variable name>:<something>}` as its own placeholder
        # so we have to trick it
        echo '${DATADOG_APM_ENABLED¤-true} && JAVA_OPTS+=" $APM_OPTS"' | sed 's/¤/:/' >> $setenv
```

I found this way more readable and has the advantage of explaining why a side trip is necessary here

---

Done, also done in the migration package